makefile

```diff
@@ -175,18 +175,16 @@ usage:
 HEADERS = $(wildcard include/*.h include/finufft/*.h) $(DUCC_HEADERS)
 
 # implicit rules for objects (note -o ensures writes to correct dir)
-%.o: %.cpp $(HEADERS)
```

Comment:
I needed to add this dependency here because otherwise it could happen that some files were compiled before `xsimd` had been installed. 
Should we make all `.o`-files depend on `$(SHEAD)` instead?

Reply:
I don't see a problem with compilation before installation for sources that *don't* depend on xsimd. But now finufft_core has xsimd dep, for alloc. I would prefer to list such dependencies as we do at l.195 - that will insure correct compilation order, right? (   Note "SHEAD" is short for "spreadintertp headers", so I would prefer that to stay specific to spreadinterp.    )

Reply:
OK, then I need to check which `.cpp` files directly and indirectly depend on `finufft_core.h`.

Reply:
I don't think you need to add them; the makefile already describes those dependencies. You merely need the one new dependency as in the next comment aroud l. 195.